CorsixTH/Lua/world.lua

```diff
@@ -1863,9 +1863,14 @@ function World:findFreeObjectNearToUse(humanoid, object_type_name, which, curren
   return object, ox, oy
 end
 
+--! Finds the most desirable room of a given room type
```

Comment:
As the "return" doc says, you're looking for a room for a humanoid rather than just one of a given type.

I'd rather just have a position to be honest instead of a humanoid, but that breaks the API.

Reply:
```suggestion
--! Finds the most desirable room of a given room type for the provided humanoid.
```